schema/bom-1.6.proto

```diff
@@ -1150,9 +1152,136 @@ message ModelCard {
+  ACTIVITY_TYPE_DATA_COLLECTION = 2;
+  // model data preparation including data cleaning, labeling and conversion.
+  ACTIVITY_TYPE_DATA_PREPARATION = 3;
+  // model building, training and generalized tuning.
+  ACTIVITY_TYPE_TRAINING = 4;
+  // refining a trained model to produce desired outputs for a given problem space.
+  ACTIVITY_TYPE_FINE_TUNING = 5;
+  // model validation including model output evaluation and testing.
+  ACTIVITY_TYPE_VALIDATION = 6;
+  // explicit model deployment to a target hosting infrastructure.
+  ACTIVITY_TYPE_DEPLOYMENT = 7;
+  // generating an output response from a hosted model from a set of inputs.
+  ACTIVITY_TYPE_INFERENCE = 8;
+}
+
+// Describes the physical provider of energy used for model development or operations.
+message EnergyProviderType {
+  // BOM unique reference to the energy provider.
+  optional string bom_ref = 1;
+  // The name of the energy provider.
+  string name = 2;
+  // A description of the energy provider.
+  optional string description = 3;
+  // TThe physical address (location) of the energy provider.BD
+  optional PostalAddressType address = 4;
+  // The energy source for the energy provider.
+  EnergySourceType energySource = 5;
+  // The energy provided by the energy source for an associated activity.
+  EnergyMeasureType energyProvided = 6;
+}
+
+enum EnergySourceType {
+  // energy source: unknown
+  // buf:lint:ignore ENUM_ZERO_VALUE_SUFFIX -- `unknown` is our fallback
+  ENERGY_SOURCE_TYPE_UNKNOWN = 0;
+  // energy source: coal
+  ENERGY_SOURCE_TYPE_COAL = 1;
+  // energy source: oil
+  ENERGY_SOURCE_TYPE_OIL = 2;
+  // energy source: natural gas
+  ENERGY_SOURCE_TYPE_NATURAL_GAS = 3;
+  // energy source: propane
+  ENERGY_SOURCE_TYPE_PROPANE = 4;
+  // energy source: nuclear
+  ENERGY_SOURCE_TYPE_NUCLEAR = 5;
+  // energy source: wind
+  ENERGY_SOURCE_TYPE_WIND = 6;
+  // energy source: solar
+  ENERGY_SOURCE_TYPE_SOLAR = 7;
+  // energy source: geothermal
+  ENERGY_SOURCE_TYPE_GEOTHERMAL = 8;
+  // energy source: hydropower
+  ENERGY_SOURCE_TYPE_HYDROPOWER = 9;
+  // energy source: other; not yet defined
+  ENERGY_SOURCE_TYPE_OTHER = 10;
+}
+
+// A measure of energy.
+message EnergyMeasureType {
+  // Quantity of energy.
+  float value = 1;
+  // Unit of energy.
+  EnergyMeasureUnitType unit = 2;
+}
+
+enum EnergyMeasureUnitType {
+  // kilowatt-hour (kWh) is the energy delivered by one kilowatt (kW) of power for one hour (h).
+  // buf:lint:ignore ENUM_ZERO_VALUE_SUFFIX -- `kWh` is our fallback
+  ENERGY_MEASURE_UNIT_TYPE_KILOWATT_HOURS = 0;
+}
+
+// A measure of carbon dioxide (CO2).
+message CO2MeasureType {
+  // Quantity of carbon dioxide (CO2).
+  float value = 1;
+  // Unit of carbon dioxide (CO2).
+  CO2MeasureUnitType unit = 2;
+}
+
+enum CO2MeasureUnitType {
+  // Tonnes (t) of carbon dioxide (CO2) equivalent (eq).
+  // buf:lint:ignore ENUM_ZERO_VALUE_SUFFIX -- `tCO2eq` is our fallback
+  CO2_MEASURE_UNIT_TYPE_TONNES_CO2_EQUIVALENT = 0;
```

Comment:
value `0` means - unset - which is equal to a "default" value.

therefore, you MUST NOT use it as such here.

Reply:
each time you think of a default here, be aware that you need to keep it constisten with the JSON and XML.
I dont see a default value for the enum there, either.

Reply:
done